docs/integrations/webhooks/debug-your-webhooks.mdx

```diff
@@ -13,23 +13,31 @@ When you or a user of your application performs certain actions, a webhook can b
 
 When you are developing on your localhost, your application is not internet facing and can't receive the webhook request. You will need to use a tool that creates a tunnel from the internet to your localhost. These tools will provide temporary or permanant URLs depending on the tool and the plan you subscribe to. Popular tools include `ngrok`, `localtunnel` and `Cloudflare Tunnel`.
 
-Below is an image demonstrating the flow from when you perform an action that triggers a webhook through until the webhook is received by your application. This flow chart will show the role of the tunnel.
-
-FLOW CHART IMAGE HERE
-
 With the Webhook flow in mind, the following areas could cause problems when trying to develop with webhooks:
 
-1. Middleware is not configured correctly for webhooks
-2. The webhook route is not correctly configured
-3. Configuration in your dashboard
-4. The tunnel configuration
-5. Verification of the webhook
-6. The logic you created to process the webhook
+1. [Middleware is not configured correctly for webhooks](#check-your-middleware-configuration)
+2. [The webhook route is not correctly configured](#test-the-route-handler-or-api-route)
+3. [Configuration in your dashboard](#check-your-configuration-in-the-clerk-dashboard)
+4. [The tunnel configuration](#debug-your-tunnel-and-webhook-delivery)
+5. [Verification of the webhook](#debug-webhook-verification)
+6. [The logic you created to process the webhook](#check-your-logic)
 
 
 ##  Check your Middleware configuration
 
-Incoming webhook events will never be signed in -- they are coming from a source outside of your application. Since they will be in a signed out state, the simplest configuration is to use `ignoredRoutes` for your webhook routes. Below is an example of making all webhook routes ignored.
+Incoming webhook events will never be signed in -- they are coming from a source outside of your application. Since they will be in a signed out state, the route should be public. The simplest configuration is to use `ignoredRoutes` for your webhook routes when using `authMiddleware` and applying no protection to the route with `clerkMiddleware`. Below is an example of making all webhook routes ignored.
```

Comment:
```suggestion
Incoming webhook events will never be signed in -- they are coming from a source outside of your application. Since they will be in a signed out state, the route should be public. 

The following example shows the recommended Middleware configuration for your webhook routes.
```